bin/virtme-prep-kdir-mods

```diff
@@ -47,10 +47,10 @@ fi
 # Delete existing .virtme_modes/lib/modules/0.0.0/modules.dep file at the beginning,
 # and regenerated by depmod at the end. So if we are interrupted during the
 # preparation of .virtme_mods folder, the next run command can correctly trigger the
-# prepararion work again.
+# preparation work again.
 
 if [ -f "$MODDIR/modules.dep" ]; then
```

Comment:
this `if` is then no longer needed if `rm` is used with `-f`.

> Get rid of interactive question when modules.dep is removed

In which distribution is this still the default behaviour?

Reply:
> this `if` is then no longer needed if `rm` is used with `-f`.
> 
> > Get rid of interactive question when modules.dep is removed
> 
> In which distribution is this still the default behaviour?

It was Ubuntu 24.04, but there can be multiple reasons for getting asked (not only if the file exists) :)